nbs/common.base_auto.ipynb

```diff
@@ -68,6 +68,7 @@
     "from ray import air, tune\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #10.    import pytorch_lightning as pl
</pre><p>It would be great if we could use <a href="https://peps.python.org/pep-0008/#:~:text=Imports%20should%20be%20grouped%20in,Local%20application%2Flibrary%20specific%20imports." target="_blank">pep8</a> style guide for imports.</p><p> </p><p> </p><pre class="ql-syntax" spellcheck="false">import torch
import pytorch_lightning as pl
from pytorch_lightning.callbacks import TQDMProgressBar
from ray import air, tune
from ray.tune.integration.pytorch_lightning import TuneReportCallback
from ray.tune.search.basic_variant import BasicVariantGenerator

</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/292/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
done